tests/testthat/_snaps/fit.md

```diff
@@ -0,0 +1,42 @@
+# mmrm works for custom optimizer
+
+    Code
+      print(fit)
+    Output
+      mmrm fit
+      
+      Formula:     FEV1 ~ ARMCD + ar1(AVISIT | SEX/USUBJID)
+      Data:        fev_data (used 537 observations from 197 subjects with maximum 4 
+      timepoints)
+      Covariance:  auto-regressive order one (4 variance parameters of 2 groups)
+      Method:      REML
+      Deviance:    4915.743
+      
+      Coefficients: 
+      (Intercept)    ARMCDTRT 
+        39.589438    5.106232 
+      
+      Model Inference Optimization:
+      Converged with code 0 and message: this is wrong
+
+# mmrm works for constructed control
+
+    Code
+      print(fit)
+    Output
+      mmrm fit
+      
+      Formula:     FEV1 ~ ARMCD + ar1(AVISIT | SEX/USUBJID)
+      Data:        fev_data (used 537 observations from 197 subjects with maximum 4 
+      timepoints)
+      Covariance:  auto-regressive order one (4 variance parameters of 2 groups)
+      Method:      REML
+      Deviance:    3855.007
+      
+      Coefficients: 
+      (Intercept)    ARMCDTRT 
+        40.290948    4.227962 
```

Comment:
This kind of snapshot (where we have numbers printed) is problematic, because on CRAN / different machines etc. we might have slightly different results. so I would highly recommend to omit these snapshots.
If you just want to test that mmrm works in certain situation, then best work with expecting silent processing, plus it converged e.g., and if comparing numbers always work with `expect_equal` or the new custom helper (which won't be applicable for this print snapshots)

Reply:
oh yes... I use the silly optimizer and thought it is fine to leave a snapshot, and in the next test I forgot that I am using a normal optimizer...